src/amo/components/InstallButtonWrapper/styles.scss

```diff
@@ -32,9 +32,21 @@
       justify-content: center;
       width: fit-content;
 
+      .GetFirefoxButton-button,
       .GetFirefoxButton-callout {
+        max-width: 100%;
+        text-align: center;
+        white-space: initial;
         width: 100%;
       }
     }
+
+    @include respond-to(large) {
+      width: auto;
+    }
+
+    @include respond-to(extraLarge) {
+      width: fit-content;
+    }
```

Comment:
do you still need those two media queries?

Reply:
We do, in fact. We need the `respond-to(large)` to deal with the issue at hand, and we don't need to re-adjust `width` at `extraLarge`, but we do need to undo the change to `white-space` at `extraLarge` or the callout and button get squished. I'm about to add a new commit with that latter change.